readthedocs/projects/tasks.py

```diff
@@ -1889,47 +1889,55 @@ def send_build_status(build_pk, commit, status):
     :param status: build status failed, pending, or success to be sent.
     """
     build = Build.objects.get(pk=build_pk)
+    provider_name = build.project.git_provider_name
 
-    try:
-        if build.project.remote_repository.account.provider == 'github':
-            service = GitHubService(
+    if provider_name == 'GitHub':
```

Comment:
This should be a constant. Also, why are we checking the name instead of `build.project.remote_repository.account.provider` like previously?

Reply:
previously we were just getting the provider name by using `build.project.remote_repository.account.provider` but now we have a better way `project.git_provider_name` as this actually checks which provider the project is from and `project.git_service_class()` get the actual class the projects needs (e.g: GitHubService) so we don't need that anymore.